src/test/unit_tests/test_s3_ops.js

```diff
@@ -94,51 +97,61 @@ mocha.describe('s3_ops', function() {
             if (bucket_type === "regular") {
                 await s3.createBucket({ Bucket: bucket_name }).promise();
                 await s3.createBucket({ Bucket: source_bucket }).promise();
-            } else if (SKIP_TEST) {
-                this.skip(); // eslint-disable-line no-invalid-this
             } else {
+                REMOTE_SOURCE_BUCKET = caching ? SOURCE_BUCKET + '-caching' : SOURCE_BUCKET;
+                REMOTE_TARGET_BUCKET = caching ? TARGET_BUCKET + '-caching' : TARGET_BUCKET;
+
                 const read_resources = [RESOURCE_NAME];
                 const write_resource = RESOURCE_NAME;
                 const read_resources1 = [RESOURCE_NAME_SOURCE];
                 const write_resource1 = RESOURCE_NAME_SOURCE;
-                const ENDPOINT = process.env.ENDPOINT ? process.env.ENDPOINT : 'https://s3.amazonaws.com';
-                const ENDPOINT_TYPE = process.env.ENDPOINT_TYPE ? process.env.ENDPOINT_TYPE : 'AWS';
+                const ENDPOINT = USE_REMOTE_ENDPOINT ? process.env.ENDPOINT : coretest.get_https_address();
+                const ENDPOINT_TYPE = USE_REMOTE_ENDPOINT ? process.env.ENDPOINT_TYPE : 'AWS';
```

Comment:
Hmm I think this should be s3-compatible for using the noobaa bucket.
This might have been the cause of the ssl certificates you had.

Reply:
This fixed the issue. Just updated the PR. Thanks!!